### PR TITLE
Fix small ERB linting issues

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

While reviewing another PR I noticed how the commit verification failed for the few issues signaled by erblint in this PR so I set out to address them 

## Added tests?

- [ ] Yes
- [x] No, and this is why: just linting issues
- [ ] I need help with writing tests

### CODE REVIEW

File: app/views/articles/tags/_sidebar.html.erb [hunk 1]
```diff
@@ -57,13 +57,29 @@
         </div>
         <% if user_signed_in? %>
           <div class="crayons-btn-group mt-3">
-            <a style="display:none;" id="tag-edit-button" data-tag="<%= @tag_model.name %>" class="crayons-btn crayons-btn--outlined mod-action-button fs-s" href="<%= tag_url(@tag_model, @page) %>/edit" style="color:<%= @tag_model.text_color_hex %>;background-color:<%= @tag_model.bg_color_hex %>">
+            <a style="display:none;"
```
Comment (rhymes) [inline app/views/articles/tags/_sidebar.html.erb:60]: Nothing has changed, it has just been split in multiple lines

File: app/views/articles/tags/_sidebar.html.erb [hunk 2]
```diff
@@ -57,13 +57,29 @@
...
+                id="tag-edit-button"
+                data-tag="<%= @tag_model.name %>"
+                class="crayons-btn crayons-btn--outlined mod-action-button fs-s"
+                href="<%= tag_url(@tag_model, @page) %>/edit"
+                style="color:<%= @tag_model.text_color_hex %>;background-color:<%= @tag_model.bg_color_hex %>">
               Edit
             </a>
-            <a style="display:none;" id="tag-admin-button" data-tag="<%= @tag_model.name %>" class="crayons-btn crayons-btn--outlined mod-action-button fs-s" href="/admin/tags/<%= @tag_model.id %>" style="color:<%= @tag_model.text_color_hex %>;background-color:<%= @tag_model.bg_color_hex %>" data-no-instant>
+            <a style="display:none;"
```
Comment (rhymes) [inline app/views/articles/tags/_sidebar.html.erb:68]: Nothing has changed, it has just been split in multiple lines

File: app/views/articles/tags/_sidebar.html.erb [hunk 3]
```diff
@@ -57,13 +57,29 @@
...
+               id="tag-admin-button"
+               data-tag="<%= @tag_model.name %>"
+               class="crayons-btn crayons-btn--outlined mod-action-button fs-s"
+               href="/admin/tags/<%= @tag_model.id %>"
+               style="color:<%= @tag_model.text_color_hex %>;background-color:<%= @tag_model.bg_color_hex %>"
+               data-no-instant>
               Admin
             </a>
-            <a style="display:none;" id="tag-mod-button" data-tag="<%= @tag_model.name %>" class="crayons-btn crayons-btn--outlined mod-action-button fs-s" href="/mod/<%= @tag_model.name %>" style="color:<%= @tag_model.text_color_hex %>;background-color:<%= @tag_model.bg_color_hex %>">
+            <a style="display:none;"
```
Comment (rhymes) [inline app/views/articles/tags/_sidebar.html.erb:77]: Nothing has changed, it has just been split in multiple lines

File: app/views/moderations/mod.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<% title "[Moderate] " + @moderatable.title %>
+<% title "[Moderate] #{@moderatable.title}" %>
```
Comment (rhymes) [inline app/views/moderations/mod.html.erb:1]: string interpolation over string concatenation

File: app/views/moderations/mod.html.erb [hunk 5]
```diff
@@ -189,7 +189,7 @@
         value="<%= rating_level %>"
         name="rating_vote_<%= rating_level %>"
         id="js__rating__vote__<%= rating_level %>"
-        class="level-rating-button article-mod-page <%= "selected" if @rating_vote&.rating == rating_level.to_f %>"
+        class="level-rating-button article-mod-page <%= "selected" if @rating_vote&.rating.to_d == rating_level.to_d %>"
```
Comment (rhymes) [inline app/views/moderations/mod.html.erb:192]: `rating_level` is an integer, float comparison is imprecise, see https://rubystyle.guide#float-comparison

File: app/views/podcasts/_form.html.erb [hunk 6]
```diff
@@ -19,7 +19,7 @@
     <div id="podcast-form-data" class="crayons-card mb-6 grid grid-flow-row gap-6 p-6">
       <div class="crayons-field crayons-field--checkbox">
         <%= check_box_tag :i_am_owner, class: "crayons-checkbox" %>
-        <%= label_tag :i_am_owner, "I am the owner of this podcast", class: "i_am_owner", class: "crayons-field__label" %>
+        <%= label_tag :i_am_owner, "I am the owner of this podcast", class: "crayons-field__label" %>
```
Comment (rhymes) [inline app/views/podcasts/_form.html.erb:22]: removed the double class

**Review by jacobherrington — APPROVED** [review]
:heart:

File: app/views/podcasts/_form.html.erb (hunk 6 above)
Comment (nickytonline) [reply app/views/podcasts/_form.html.erb:22]: Out of curiosity, prior to your fix @rhymes, does the last `class` property overwrite the previous one or do they get concatenated?

Comment (rhymes) [reply app/views/podcasts/_form.html.erb:22]: @nickytonline it gets overwritten, `crayons-field__label` is the only one that survives, see in the HTML of https://dev.to/podcasts/new

**Review by nickytonline — APPROVED** [review]
Thanks for the cleanup @rhymes!

**Pull request merged**